dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -53,6 +53,7 @@ static struct Context {
 
 static const char *LOG_TAG = "DatadogNdkCrashReporter";
 static pthread_mutex_t handler_mutex = PTHREAD_MUTEX_INITIALIZER;
+static uint8_t tracking_consent = 0; // 0 - PENDING, 1 - GRANTED, 2 - NOT-GRANTED
```

Comment:
I could use an enum here. Not sure if it is needed though as it will be extra conversion work for nothing.

Reply:
Alternatively we could just do this:
```suggestion
static uint8_t tracking_consent_pending = 0;
static uint8_t tracking_consent_granted = 1;
static uint8_t tracking_consent_not_granted = 2;
```